modules/odc-access-web-terminal.adoc

```diff
@@ -1,13 +1,15 @@
 // Module included in the following assemblies:
 //
-// web_console/odc-about-web-terminal.adoc
+// web_console/odc-web-terminal.adoc
+
 :_content-type: PROCEDURE
-[id="odc-using-web-terminal_{context}"]
-= Using the web terminal
+[id="odc-access-web-terminal_{context}"]
+= Accessing the web terminal
 
-After the Web Terminal Operator is installed, you can use the web terminal as follows:
+After the {web-terminal-op} is installed, you can access the web terminal as follows:
 
 . To launch the web terminal, click the command line terminal icon (image:odc-wto-icon.png[title="wto icon"]) on the upper right of the console. A web terminal instance is displayed in the *Command line terminal* pane. This instance is automatically logged in with your credentials.
```

Comment:
This step should be rewritten to follow the ISG guidance for referring to the location of UI elements: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements.

> When you refer to the location of an element in a graphical user interface, do not use sensory words, such as top, bottom, right, or left.

Reply:
Opened a follow up Jira: https://issues.redhat.com/browse/RHDEVDOCS-5304